docs/advanced/peering/peering-via-cr.md

```diff
@@ -0,0 +1,428 @@
+I1107 10:05:29.355741       1 reflector.go:163] [cl02] Reflection of authentication.liqo.io/v1beta1, Resource=resourceslices correctly started
+```
+
+### Summary of authentication configuration
+
+To sum up, to set up the authentication, **on the cluster provider** you will need to:
+
+- Create the credentials to be used by the consumer
+- Bind the credentials to the `liqo-remote-controlplane` role
+- Create a `Tenant` resource for the consumer
+
+While, **on the consumer side**:
+
+- Create a new secret containing the kubeconfig with the credentials to access to the cluster provider
+
+## Declarative configuration of namespace offloading
+
+While offloading is independent from the network, which means that it is possible to negotiate resources and configure a namespace offloading without the inter-cluster network enabled, **a [working authentication configuration](#declarative-configuration-of-clusters-authentication) is a pre-requisite to enable offloading**.
+
+### Ask for resources: configure a ResourceSlice
+
+The `ResourceSlice` resource is the one to be created to ask a cluster provider for resources.
+The following is an example of `ResourceSlice`:
+
+```{code} yaml
+apiVersion: authentication.liqo.io/v1beta1
+kind: ResourceSlice
+metadata:
+  annotations:
+    liqo.io/create-virtual-node: "true"
+  creationTimestamp: null
+  labels:
+    liqo.io/remote-cluster-id: <PROVIDER_CLUSTER_ID>
+    liqo.io/remoteID: <PROVIDER_CLUSTER_ID>
+  name: test
+  namespace: liqo-tenant-cl02
+spec:
+  class: default
+  providerClusterID: <PROVIDER_CLUSTER_ID>
+  resources:
+    cpu: 20
+    ram  128Gi
+```
+
+With the configuration above, once the resources are accepted from the provider side, a new (virtual) node, impersonating the provider cluster, will make available the requested resorces on the consumer cluster.
+
+To know more about `ResourceSlice` and `VirtualNode` check [this section of the documentation](./offloading-in-depth.md#create-resourceslice).
+
+### Enable offloading and K8s resources availability on remote clusters
+
+By default, the virtual nodes are not eligible for task scheduling, unless offloading is enabled for the namespace where pod is running.
+To do so, a `NamespaceOffloading`, like the following, should be created:
+
+```{code} yaml
+apiVersion: offloading.liqo.io/v1beta1
+kind: NamespaceOffloading
+metadata:
+  name: offloading
+  namespace: demo
+spec:
+  clusterSelector:
+    nodeSelectorTerms: []
+  namespaceMappingStrategy: DefaultName
+  podOffloadingStrategy: LocalAndRemote
+```
+
+The `NamespaceOffloading` resource should be created in the namespace that we want to extend on the remote clusters.
+
+For example, the resource above, extends the `demo` namespace on all the configured provider clusters.
+As the policy is `LocalAndRemote` the pod will be executed locally or remotely and it depends on the choice made by the the Kubernetes scheduler.
```

Comment:
```suggestion
Since the `podOffloadingStrategy` policy is `LocalAndRemote`, the a new pod could be executed either locally or remotely depending on the choice made by the vanilla Kubernetes scheduler (e.g., if the remote virtual node is plenty of free resources, it may be preferred against local nodes that are already used by other pods).
```